meinberlin/templates/a4dashboard/includes/nav_modules_item.html

```diff
@@ -62,15 +62,20 @@
 </div>
 
 <div class="modal" tabindex="-1" role="dialog" id="delete-modal{{ module_menu.module.pk }}" aria-labelledby="delete-modal-label">
-    <div class="modal-dialog modal-lg" role="document">
+    <div class="modal-dialog modal-dialog-centered modal-lg" role="document">
         <div class="modal-content">
             <div class="modal-header">
-                <h2 class="modal-title u-first-heading" id="delete-modal-label">{% translate "Are you sure you want to delete your module?" %}</h2>
+                <h1 class="modal-title u-first-heading" id="delete-modal-label">{% translate 'Delete' %}</h1>
                 <button type="button" class="close" data-bs-dismiss="modal" aria-label="{% translate 'Close'%}">
                     <span aria-hidden="true">&times;</span>
                 </button>
             </div>
             <div class="modal-body">
+                <div class="u-spacer-bottom">
```

Comment:
also for some reason i'm seeing this i dashboard?! 
![screenshot(4)](https://user-images.githubusercontent.com/5871230/215840927-573ae187-dc3d-4529-b3a4-3a7ce4506523.png)

Reply:
> u-spacer-bottom can also be added to modal-body or we can change the variable if makes sense everywhere

no because in case there is `partials.description` and `partials.body` we want that spacing for the description. That's why it is separated.

Reply:
> also for some reason i'm seeing this i dashboard?!
> ![screenshot(4)](https://user-images.githubusercontent.com/5871230/215840927-573ae187-dc3d-4529-b3a4-3a7ce4506523.png)

this doesn't happen to me, maybe related with re-link browser cache from above?